bash/blddist

```diff
@@ -7,71 +7,68 @@ set -e                                  # exit on all errors
 cd $(dirname $0)                        # set to directory of script
 cd ..                                   # cd back to root       
 
-root_dir=$(pwd)
-version_tag=$(git describe)
-dist_folder=dist
+ROOT_DIR=$(pwd)
+VERSION="$(git describe --tags)"
+VERSION="${VERSION//./_}"
```

Comment:
Question only. Is this line here (also line 18 of build.yml) because of issue 518 zc390.java fails when directory name contains a period? If yes, a potential fix exists. See issue 518 comment added Oct 7.

Reply:
I have added your fix and removed the workaround.